db/migrate/20230706081829_remove_user_id_from_posts.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUserIdFromPosts < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :posts, :user_id, :integer
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to remove this file, there is no `user_id` column in the `posts` table, this produces an error when creating the migrations.